src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.html

```diff
@@ -7,129 +7,91 @@
+          <p class="days-remaining m-0">{{getRemainigDays(profile?.responseDeadline)}}</p>
+          <a [href]="createCalendarEvent()" class="icon-circle" target="_blank">
+            <i class="fa fa-calendar-plus-o" aria-hidden="true"></i>
+          </a>
         </div>
-
+        <button *ngIf="consultationStatus === 'active'" class="btn-submit mt-3 active" (click)='stepNext(profile?.respondedOn)' data-event="lead">
+          <p class="m-0">
+            <span>
+              <picture class="citizen-logo">
+                <source srcset="assets/images/Vector.webp" type="image/webp">
+                <source srcset="assets/images/Vector.jpg" type="image/jpeg">
+                <img src="assets/images/Vector.jpg" />
+              </picture>
+            </span>
+            {{'SUBMIT RESPONSE' | translate}}
+          </p>
+          <span><i class="fa fa-angle-right"></i></span>
+        </button>
       </div>
-      <div *ngIf="summaryData" class="report-download-block">
+      <div *ngIf="profile?.respondedOn && !showResponseCreation" class="response-submitted-block">
         <div>
-          <button  class="btn-submit mt-3  active" (click)='downloadReport()'>
-            <p class="m-0">
-              <span><i class="fa fa-download"></i></span>
-              <span>{{'Download report as pdf' | translate}}</span>
-            </p>
-          </button>
+          <span class="success-text mr-1"><i class="fa fa-check"></i></span>
         </div>
+        <div>
+          <p class="mb-1 success-text">
+            {{'RESPONSE SUBMITTED' | translate}}
+          </p>
+          <p class="success-date m-0">{{convertDateFormat(profile?.respondedOn)}}</p>
+        </div>
+      </div>
+    </div>
+    <div class="share-block" *ngIf="profile?.visibility === 'public_consultation'">
+      <div class="spread-btn mt-3">
+        <button class="btn-circular" (click)="showShareOptions = !showShareOptions">
+          <span class="mr-2 clr-blue"><i class="fa fa-share"></i></span>
+          <span class="blue-text bold-text">{{'Spread the word' | translate}}</span>
+        </button>
       </div>
+    </div>
+    <hr class="divider-one" />
   </div>
 
-  <hr class="divider" />
-
   <div class="c-profile-card__bottom-section">
     <div class="d-flex justify-content-between mb-3">
       <p class="count-text m-0">{{'TOTAL RESPONSES' | translate}}</p>
       <p class="count-number bold-text text-right m-0">
         {{profile?.consultationResponsesCount ? profile?.consultationResponsesCount : 0}}
       </p>
     </div>
-    <div
-      class="d-flex justify-content-between mb-3"
-      *ngIf="profile?.reviewType && profile?.reviewType !== 'policy'"
-    >
+    <div class="d-flex justify-content-between mb-3" *ngIf="profile?.reviewType && profile?.reviewType !== 'policy'">
       <p class="count-text m-0">{{'TOTAL PUBLIC RESPONSES' | translate}}</p>
-      <p class="count-number bold-text text-right m-0" *ngIf="!summaryData; else summaryDataCount">
```

Comment:
Why was this condition removed?

Reply:
I notice this bro, while formating it would have been done.

Reply:
@Bandinikhil not clear what you meant bro... If you removed it by mistake it needs to be added back

Reply:
It is been added bro.